internal/pkg/scorecard/plugins/resource_handler.go

```diff
@@ -179,6 +179,13 @@ func getPodFromDeployment(depName, namespace string) (pod *v1.Pod, err error) {
 		}
 		// Make sure the pods exist. There should only be 1 pod per deployment.
 		if len(pods.Items) == 1 {
+			//make sure both containers are Running
```

Comment:
Nit: since who knows, we might change how the operator pod deployment works in the future.
```suggestion
			// Make sure all containers are Running
```

Also, from a logic standpoint, now might be a good time to refactor this function to handle the `false` return conditions first. Something like:

```go
// Make sure the pods exist. There should only be 1 pod per deployment.
if len(pods.Items) > 1 {
	log.Debug("Operator deployment has more than 1 pod")
	return false, nil
}

// If the pod has a deletion timestamp, it is the old pod; wait for
// pod with no deletion timestamp
if pods.Items[0].GetDeletionTimestamp() != nil {
	return false, nil
}

// Make sure all containers are Running
for _, containerStatus := range pods.Items[0].Status.ContainerStatuses {
	if containerStatus.State.Running == nil {
		return false, nil
	}
}

pod = &pods.Items[0]
return true, nil
```